packages/@aws-cdk/aws-ec2/lib/volume.ts

```diff
@@ -642,7 +641,8 @@ export class Volume extends VolumeBase {
   }
 
   protected validateProps(props: VolumeProps) {
-    if (!Token.isUnresolved(props.availabilityZone) && !/^[a-z]{2}-[a-z]+-[1-9]+[a-z]$/.test(props.availabilityZone)) {
+    // Note: dummy1[a-z] comes from the DUMMY_VPC_PROPS in vpc.ts. This will be encountered on the first run of a fresh app.
```

Comment:
see https://github.com/aws/aws-cdk/issues/9081#issuecomment-659521261

Reply:
@NetaNir New revision posted that removes the input validation of the AZ entirely.